Shared/Samples/Identify raster cell/IdentifyRasterCellView.swift

```diff
@@ -0,0 +1,252 @@
+                    calloutPlacement = nil
+                }
+            }
+        }
+        
+        /// Identifies the raster cell for a given screen point on the raster layer.
+        /// - Parameters:
+        ///   - screenPoint: The screen point corresponding to a raster cell.
+        ///   - proxy: The proxy used to identify the screen point on the raster layer.
+        /// - Returns: The first raster cell found in the identify result.
+        private func rasterCell(for screenPoint: CGPoint, using proxy: MapViewProxy) async -> RasterCell? {
+            do {
+                guard let rasterLayer else {
+                    throw CustomError.message("Raster layer is not initialized.")
+                }
+                
+                // Identify the screen point on the raster layer using the map view proxy.
+                let identifyResult = try await proxy.identify(on: rasterLayer, screenPoint: screenPoint, tolerance: 1)
+                
+                // Get the first raster cell from the identify result.
+                let rasterCell = identifyResult.geoElements.first(where: { $0 is RasterCell })
+                return rasterCell as? RasterCell
+            } catch {
+                self.error = error
+                return nil
+            }
+        }
+        
+        /// Updates the location of the callout placement to a given screen point.
+        /// - Parameters:
+        ///   - screenPoint: The screen point at which to place the callout.
+        ///   - proxy: The proxy used to convert the screen point to a map point.
+        ///   - offsetted: A Boolean value that indicates whether to offset the callout.
+        private func updateCalloutPlacement(to screenPoint: CGPoint, using proxy: MapViewProxy, offsetted: Bool) {
+            // Create an offset to offset the callout if needed, e.g. the magnifier is showing.
+            let offset = offsetted ? CGPoint(x: 0, y: -70) : .zero
+            
+            // Disable the placement animation when there is an offset.
+            // This is done to prevent the callout from lagging when the magnifier is moved.
+            calloutShouldAnimate = !offsetted
+            
+            // Get the map location of the screen point from the map view proxy.
+            if let location = proxy.location(fromScreenPoint: screenPoint) {
+                calloutPlacement = .location(location, offset: offset)
+            }
+        }
+        
+        /// Updates the text shown in the callout using the attributes and coordinates of a given raster cell.
+        /// - Parameter cell: The raster cell to create the text from.
+        private func updateCalloutText(using cell: RasterCell) {
+            // Create the attributes text using the attributes of the raster cell.
+            let attributes = cell.attributes.map { "\($0.key): \($0.value)" }.sorted(by: >).joined(separator: "\n")
```

Comment:
This line is pretty hard to read, we can clean that up:
```suggestion
            let attributes = cell.attributes
                .map { "\($0.key): \($0.value)" }
                .sorted(by: >)
                .joined(separator: "\n")
```